indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +437,12 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    @make_interceptable
+    def can_see(self, user, is_management):
```

Comment:
For readability it's better to always pass this as a kwarg:

```suggestion
    def can_see(self, user, *, is_management):
```